.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "trailingComma": "all",
+  "singleQuote": true,
+  "printWidth": 120,
+  "tabWidth": 4
```

Comment:
2 space indent!

Reply:
No blind copy and paste! gotcha, this is fixed